stages/org.osbuild.pki.update-ca-trust.meta.json

```diff
@@ -0,0 +1,10 @@
+{
+  "summary": "Insert CA files and update CA trust store",
```

Comment:
Unless I'm mistaken the summary/description does not longer quite match(?). This stage just updates the ca-trust store now, the adding needs to be done via inline source and copy. So probably good to update and explain that a bit too :)